apps/web/screens/Article/components/ArticleChatPanel/config.ts

```diff
@@ -0,0 +1,353 @@
+        sessionStorage.clear()
+      }
+      sessionStorage.setItem('syslumennChatPanelCategory', 'endokuskirteini')
+
+      // eslint-disable-next-line @typescript-eslint/no-explicit-any
+      const preSendhandler = (event: any) => {
+        event.data.context.skills[
+          'main skill'
+        ].user_defined.category_endokuskirteini = true
+      }
+
+      instance.on({ type: 'pre:send', handler: preSendhandler })
+
+      instance.updateHomeScreenConfig({
+        is_on: false,
+      })
+    },
+  },
+  // Þinglýsing skjala
+  // https://app.contentful.com/spaces/8k0h54kbe6bj/entries/2evftN0gIe78zSEYLMB0aX
+  '2evftN0gIe78zSEYLMB0aX': {
+    integrationID: '0c96e8fb-d4dc-420e-97db-18b0f8bb4e3f',
+    region: 'eu-gb',
+    serviceInstanceID: 'bc3d8312-d862-4750-b8bf-529db282050a',
+    showLauncher: false,
+    carbonTheme: 'g10',
+    cssVariables: {
+      'BASE-font-family': '"IBM Plex Sans", "Open Sans", Arial, sans-serif',
+    },
+    languagePack: syslumennLanguagePack,
+    onLoad: (instance) => {
+      const category = sessionStorage.getItem('syslumennChatPanelCategory')
+      if (category !== 'thinglysing') {
+        sessionStorage.clear()
+      }
+      sessionStorage.setItem('syslumennChatPanelCategory', 'thinglysing')
+
+      // eslint-disable-next-line @typescript-eslint/no-explicit-any
+      const preSendhandler = (event: any) => {
+        event.data.context.skills[
+          'main skill'
+        ].user_defined.category_thinglysing = true
+      }
+
+      instance.on({ type: 'pre:send', handler: preSendhandler })
+
+      instance.updateHomeScreenConfig({
+        is_on: false,
+      })
+    },
+  },
+  // Vegabréf, almennar upplýsingar
+  // https://app.contentful.com/spaces/8k0h54kbe6bj/entries/7Egh4yMfC48dDgceeBrWSB
+  '7Egh4yMfC48dDgceeBrWSB': {
+    integrationID: '0c96e8fb-d4dc-420e-97db-18b0f8bb4e3f',
+    region: 'eu-gb',
+    serviceInstanceID: 'bc3d8312-d862-4750-b8bf-529db282050a',
+    showLauncher: false,
+    carbonTheme: 'g10',
+    cssVariables: {
+      'BASE-font-family': '"IBM Plex Sans", "Open Sans", Arial, sans-serif',
+    },
+    languagePack: syslumennLanguagePack,
+    onLoad: (instance) => {
+      const category = sessionStorage.getItem('syslumennChatPanelCategory')
+      if (category !== 'vegabref') {
+        sessionStorage.clear()
+      }
+      sessionStorage.setItem('syslumennChatPanelCategory', 'vegabref')
+
+      // eslint-disable-next-line @typescript-eslint/no-explicit-any
+      const preSendhandler = (event: any) => {
+        event.data.context.skills[
+          'main skill'
+        ].user_defined.category_vegabref = true
+      }
+
+      instance.on({ type: 'pre:send', handler: preSendhandler })
+
+      instance.updateHomeScreenConfig({
+        is_on: false,
+      })
+    },
+  },
+  // Sakavottorð til einstaklinga
+  // https://app.contentful.com/spaces/8k0h54kbe6bj/entries/gzKeBtRl57SzRmgUzHR3u
+  gzKeBtRl57SzRmgUzHR3u: {
```

Comment:
```suggestion
  'gzKeBtRl57SzRmgUzHR3u': {
```
Quotes for consistency

Reply:
The quotes got removed by the formatter